packages/theme/src/styles/default-theme.css

```diff
@@ -90,6 +90,7 @@
 
   /* Shadows */
   --swui-shadow-modal: rgba(0, 0, 0, 0.2) 0px 0px 10px 4px;
+  --swui-shadow-modal-footer: -1px 0px 25px -2px rgba(0, 0, 0, 0.2);
```

Comment:
This is too specific. These should be based on roles, not specific use-case.
I suggest you just use `--swui-shadow-modal` instead.

Reply:
Fixed!